plateau/core/common_metadata.py

```diff
@@ -146,6 +146,33 @@ def set(self, i, field):
     set.__doc__ = pa.Schema.set.__doc__
 
 
+def gen_metadata(schema: SchemaWrapper) -> dict[str, Any]:
+    assert isinstance(schema.internal(), pa.Schema), (
+        "Internal schema must be a pyarrow schema"
+    )
+
+    pandas_metadata = {
+        "columns": [],
+        "index_columns": [],  # this function is only called if no pandas metadata is present, ergo we do not have any index columns
+        "pandas_version": pd.__version__,
```

Comment:
Please hardcode the pandas version here. We are not using `pandas` at all but hardcode the logic that pandes uses at the time of writing.

Reply:
[apparently](https://pandas.pydata.org/docs/development/developer.html), the "logic" uses at the time of writing is to just use the current version. So should I keep the `pd.__version__`?

Reply:
Time of writing is the "time of writing this PR"
```suggestion
        "pandas_version": "2.0.0",
```